google/cloud/bigquery/_helpers.py

```diff
@@ -82,7 +82,18 @@ def _timestamp_from_json(value, field):
     """Coerce 'value' to a datetime, if set or not nullable."""
     if _not_null(value, field):
         # value will be a float in seconds, to microsecond precision, in UTC.
-        return _datetime_from_microseconds(1e6 * float(value))
```

Comment:
This will need to be modified to parse from an rfc timestamp string.

Reply:
Sorry, i didn't get the point. Before passing `params["formatOptions.useInt64Timestamp"] = True` in `client.list_row`, received a value in string with decimal  in `_timestamp_from_json`  method , but after passing it, received a value in string without decimal and call the method `_datetime_from_microseconds(int(value))` which returns datetime object.

Reply:
Oh I see. Yes, that's expected.

Reply:
In this case, we should update the comment.

Also, are there cases where floating point values are still passed in? If not, we should clean this up now. If so, let's add a TODO to identify those cases and file an issue to clean them up.